src/components/member-card/index.js

```diff
@@ -7,16 +7,32 @@ import PropTypes from 'prop-types';
 import ShowSkills from '@components/member-card/show-skills';
 import SuperUserOptions from '@components/member-card/super-user-options';
 
-const Card = ({ developerInfo, isOptionKey }) => {
+const Card = ({ developerInfo, isOptionKey, colorCombination }) => {
   const { query } = useRouter() || { query: { dev: false } };
   const { dev } = query;
   const { username, first_name, last_name, img_url, isMember } = developerInfo;
+  const { color_primary, color_secondary } = colorCombination;
+  const cardRef = useRef();
+  const textRef = useRef();
+
   const socialMedia = [
     'twitter_id',
     'github_id',
     'linkedin_id',
     'instagram_id',
   ];
+
+  useEffect(() => {
+    const cardElement = cardRef.current;
```

Comment:
I don't think use of 2 useRef will be the better option. Can we make use of inline style instead of writing style in abstract way?

Reply:
yeah looks like an overkill, thanks for the suggestion made the change